tools/sklearn_tools/eden_converter.xml

```diff
@@ -0,0 +1,123 @@
+<tool id="eden_converter" name="Convert" version="@VERSION@">
```

Comment:
This is the sequence specific converter, right? If you plan to have an additional one for molecules we need to change the id and make it more explicit.